lesson-6/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link href="style/style.css"rel="stylesheet">  
+</head>
+<body>
+ <div class="rectangle52"> 
+  <header>
+   <div class="vector">    
+     <img src="style/img/logo_vector.svg" alt="logo vector">   
+   </div>
+   <div class="right-part">
+     <div class="stays">
+       <p class="text-block stays">Stays</p>
+       <hr>
+     </div>
+     <p class="text-block attractions">Attractions</p>
+     <input type="image" id="night" src="style/img/Night.svg" alt="Night">
+     <input type="image" id="accountCircle" src="style/img/AccountCircle.svg" alt="AccountCircle">
+    </div>
+  </header>
+  <section>
+   <h1 class="text-discover">Discover stays <br> to live, work or just relax</h1>
+   <div class="search">
```

Comment:
There is no "Your destination or hotel name" and "Check-in — Check-out"  before inputs